examples/carousel/js/carousel-tablist.js

```diff
@@ -0,0 +1,373 @@
+  }
+
+  // Start rotation
+  setTimeout(this.rotateSlides.bind(this), this.timeInterval);
+
+}
+
+CarouselTablist.prototype.centerTablistControls = function () {
+  width1 = this.tablistNode.getBoundingClientRect().width;
+  width2 = this.containerNode.getBoundingClientRect().width;
+  width3 = this.pauseButtonNode.getBoundingClientRect().width;
+  this.tablistNode.style.left = (((width2-width1)/2) - (3*width3/2)) +'px';
+}
+
+
+CarouselTablist.prototype.getTabpanelNode = function (tabNode) {
+  var index = this.tabNodes.indexOf(tabNode);
+  return this.tabpanelNodes[index];
+}
+
+CarouselTablist.prototype.hideTabpanel = function (tabNode) {
+  tabNode.setAttribute('aria-selected', 'false');
+  tabNode.setAttribute('tabindex', '-1');
+
+  var tabpanelNode = this.getTabpanelNode(tabNode);
+
+  if (tabpanelNode) {
+    tabpanelNode.classList.remove('active');
+  }
+}
+
+CarouselTablist.prototype.showTabpanel = function (tabNode, moveFocus) {
+  if (typeof moveFocus !== 'boolean') {
+    moveFocus = false;
+  }
+
+  tabNode.setAttribute('aria-selected', 'true');
+  tabNode.removeAttribute('tabindex');
+
+  var tabpanelNode = this.getTabpanelNode(tabNode);
+
+  if (tabpanelNode) {
+    tabpanelNode.classList.add('active');
+  }
+
+  if (moveFocus) {
+    tabNode.focus();
+  }
+}
+
+CarouselTablist.prototype.setSelectedTab = function (newTab, moveFocus) {
+  if (typeof moveFocus != 'boolean') {
+    moveFocus = false;
+  }
+
+  for (var i = 0; i < this.tabNodes.length; i++) {
+    this.hideTabpanel(this.tabNodes[i]);
+  }
+
+  this.currentTabNode = newTab;
+  this.showTabpanel(newTab, moveFocus);
+}
+
+CarouselTablist.prototype.setSelectedToPreviousTab = function (currentTabNode, moveFocus) {
+  if (typeof moveFocus != 'boolean') {
+    moveFocus = false;
+  }
+
+  var index;
+
+  if (typeof currentTabNode !== 'object') {
+    currentTabNode = this.currentTabNode;
+  }
+
+  if (currentTabNode === this.firstTabNode) {
+    this.setSelectedTab(this.lastTabNode, moveFocus);
+  }
+  else {
+    index = this.tabNodes.indexOf(currentTabNode);
+    this.setSelectedTab(this.tabNodes[index - 1], moveFocus);
+  }
+}
+
+CarouselTablist.prototype.setSelectedToNextTab = function (currentTabNode, moveFocus) {
+  if (typeof moveFocus != 'boolean') {
+    moveFocus = false;
+  }
+
+  var index;
+
+  if (typeof currentTabNode !== 'object') {
+    currentTabNode = this.currentTabNode;
+  }
+
+  if (currentTabNode === this.lastTabNode) {
+    this.setSelectedTab(this.firstTabNode, moveFocus);
+  }
+  else {
+    index = this.tabNodes.indexOf(currentTabNode);
+    this.setSelectedTab(this.tabNodes[index + 1], moveFocus);
+  }
+}
+
+CarouselTablist.prototype.setSelectedTofirstTabNode = function (moveFocus) {
+  if (typeof moveFocus != 'boolean') {
+    moveFocus = false;
+  }
+  this.setSelectedTab(this.firstTabNode, moveFocus);
+}
+
+CarouselTablist.prototype.setSelectedTolastTabNode = function (moveFocus) {
+  if (typeof moveFocus != 'boolean') {
```

Comment:
```suggestion
  if (typeof moveFocus !== 'boolean') {
```

Reply:
It looks like there are actually 5 instances of a non-strict equals with `boolean`; it'd be nice to convert them all into a strict equals check for consistency

Reply:
@smhigley 

fixed